src/UTF8Util.hpp

```diff
@@ -75,25 +75,7 @@ class OPENCC_EXPORT UTF8Util {
   * Returns the length in byte for the previous UTF8 character.
   */
   static size_t PrevCharLength(const char* str) {
-    {
```

Comment:
I believe these are an optimization based on likeliness character's encoded length are in the converted text.
Most frequently used Chinese characters are encoded in 3 bytes; followed by 1-byte ASCII. 4 to 6-byte characters are rare.